LimeBean.Tests/BeanTests.cs

```diff
@@ -114,6 +114,23 @@ public class BeanTests {
         }
 
         [Fact]
+        public void Columns() {
+            var bean = new Bean();
+            bean["a"] = 1;
+            bean["b"] = 2;
+            bean["c"] = null;
+
+            List<string> expect = new List<string>(new[] { "a", "b", "c" });
+            
+            Assert.Equal(3, bean.Columns.Count());
```

Comment:
By the way, there is a utility `AssertExtensions.Equivalent` to compare unordered collections, see https://github.com/Nick-Lucas/LimeBean/blob/master/LimeBean.Tests/AssertExtensions.cs